modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseUpgradeCheck.java

```diff
@@ -160,6 +181,6 @@ protected boolean isValidExtension(String fileName) {
 	private static final Pattern _copyrightPattern = Pattern.compile(
 		"(<%--\\s*(\\/\\*)+(\\n|.)*(\\*\\/)+\\s*--%>)");
 	private static final Pattern _includesPattern = Pattern.compile(
-		"(<%@\\s*include\\s*(.+)%>\\s*)+", Pattern.MULTILINE);
+		"<%@\\s*include\\s*file\\s*=[\"/\\w\\.]+\\s*%>", Pattern.MULTILINE);
```

Comment:
@nicolas-sss, shouldn't we use `\\s+` instead of `\\s*` before `file`?

After all, we should have, at least, 1 blank space separating the words `include` and `file`, does it make sense to you?